eq-author/src/App/page/Design/Validation/DateValidation/PositionPicker.js

```diff
@@ -2,9 +2,9 @@ import React from "react";
 import PropTypes from "prop-types";
 import { RelativePositionSelect } from "./components";
 
-const RELATIVE_POSITIONS = ["before", "after"];
+const RELATIVE_POSITIONS = ["Before", "After"];
```

Comment:
Inside the map you're doing `position.toLowerCase()`, so I think we should keep it consistent and make the `RELATIVE_POSITIONS` be all lower case too. Or, will it break something?

```suggestion
const RELATIVE_POSITIONS = ["before", "after"];
```

Reply:
Just for clarity sake, the lowercase is causing the issue with the relative position